plugins/janus_videoroom.c

```diff
@@ -4237,6 +4252,7 @@ void janus_videoroom_incoming_rtp(janus_plugin_session *handle, int video, char
 					janus_mutex_lock(&videoroom->mutex);
 					json_t *event = json_object();
 					json_object_set_new(event, "videoroom", json_string(participant->talking ? "talking" : "stopped-talking"));
+					json_object_set_new(event, "audio-level-dBov-avg", json_real(audio_dBov_avg));
```

Comment:
Semantically, this should be added after "id": not really important, but since we encode properties in order and in other parts of the code we do the same, I'd prefer it to be that way, for style.

Reply:
done.